packages/fxa-admin-server/package.json

```diff
@@ -4,10 +4,10 @@
   "description": "FxA GraphQL Admin Server",
   "scripts": {
     "prebuild": "rimraf dist",
-    "build": "yarn nest build && cp ./src/config/*.json ./dist/config",
+    "build": "yarn prebuild && yarn nest build && cp ./src/config/*.json ./dist/config",
```

Comment:
I thought prebuild should run when `yarn build` was executed, but it doesn't seem to be the case...